targets/lantiq-xrx200

```diff
@@ -1,3 +1,14 @@
 device('avm-fritz-box-7412', 'avm_fritz7412', {
         factory = false,
 })
+
+-- TP-Link
+
+device('tp-link-td-w8970', 'tplink_tdw8970', {
+	broken = true, -- We have not yet made a decision on how to handle hybrid LAN/WAN ports.
+})
+
+device('tp-link-td-w8980', 'tplink_tdw8980', {
+	aliases = {'tp-link-td-w9980'},
+	broken = true, -- 5GHz unsupported
+})
```

Comment:
GitHub marks a missing newline at end-of-file, please check if there was one before and in the other target files and if yes, please add the newline (newline != empty line)